contract-tests/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Dockerfile for running the contract test service. At this point we have
+# already compiled the code, so we only need to copy the executable.
```

Comment:
The reason I've set up the CI steps to build the test service first, and then copy it into the Docker container— instead of setting up the Dockerfile to build the test service inside of the container— is that it's a bit faster that way, because we've already built the eventsource library so it's already downloaded the necessary modules.